src/components/card-grid/card-grid.jsx

```diff
@@ -13,8 +13,8 @@ export default class CardGrid extends React.Component {
     this.state = this.modelInitStateFromDataProp(props.data);
   }
 
-  componentWillReceiveProps(nextProps) {
-    const state = this.modelInitStateFromDataProp(nextProps.data);
+  UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
```

Comment:
do you have any background info on this change?

Reply:
@thescientist13 
I knew this might have come up. When running `npm run build`, I'm seeing the following error due to the installed version of React. I had some trouble revisiting this project after a while so wiped the `node_modules` and `yarn.lock`. When the dust settled, this error was being thrown.

```
ERROR in ./components/card-grid/card-grid.jsx

/Users/paul/Projects/DevLab2425/repos/ProvidenceGeeks/website-frontend/src/components/card-grid/card-grid.jsx
  16:3  error  componentWillReceiveProps is deprecated since React 16.9.0, use UNSAFE_componentWillReceiveProps instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops. Use https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles to automatically update your components  react/no-deprecated

✖ 1 problem (1 error, 0 warnings)
```

Reply:
Hey @DevLab2425 , thanks for the information, very helpful.  

Can you confirm, are you using Yarn to install dependencies or NPM?  I can understand blowing out _node_modules_ but you shouldn't blow out the lock file.  That is what pins all the versions of our dependences, coincidently to avoid the "works on my machine" problem.  :)

My guess is blowing out the lock file had you pull a newer version of React than what the project was developed against.

Reply:
🤦‍♂ 
Yep. The upstream `yarn.lock` installed `react@16.8.6`. No warning or error being thrown now. I'll revert the commit. (The other PRs too)